charts/nexus3/values.yaml

```diff
@@ -392,7 +392,8 @@ config:
     groupMemberFormat:
   # -- Blob store configuration.
   blobStores: []
-  # Reference the Nexus Blob store REST API for supported types and expected request body structures of each
+  # -- Reference the Nexus Blob store REST API for supported types and expected request body structures of each
+  # -- You can find the API reference on an existing Nexus installation at Administration section under System → API
```

Comment:
This isn't a helm-docs comment, so adding the `--` will just break the docs. You need to work with the existing comment structure, for example see below.

```yaml
# -- Blob store configuration; based on the REST API.

Reply:
But the official documentation says differently: https://helm.sh/docs/chart_best_practices/values/
I don't know if multiline is support. Hope that fits now.

Reply:
@mreiche best practices alway have a contradiction, and the best practices for a system (e.g. a collection of Helm charts and the adjacent tooling) generally supercede those of the components. In this case, as is the case for all the charts in this repo, the supported values can be loosely defined YAML objects with a single entry in the README; this is very much best practice but wouldn't be present in a contrived example such as is in the Helm docs.

Reply:
I'm not familiar with writing helm charts this professional. Mea culpa.